src/main/modules/nunjucks/index.ts

```diff
@@ -22,9 +22,9 @@ export class Nunjucks {
     });
 
     env.addGlobal('getError', function (fieldName: string): { text?: string } | boolean {
-      const { form, sessionErrors, errors, formState } = this.ctx;
+      const { form, sessionErrors, errors } = this.ctx;
 
-      const hasMoreThanTwoFields = new Form(form, formState).getFieldNames().size >= 2;
+      const hasMoreThanTwoFields = new Form(form.fields).getFieldNames().size >= 2;
```

Comment:
I had no idea a form gets constructed here, that's pretty horrible but we'll have to look at it later